### PR TITLE
fix: README consistency & typings

### PR DESCRIPTION
**Pull request by ember-ana**
to enforce a similar style for all plugins: copied from functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
-# klasa-rawevents-plugin
+# klasa-raw-events
```
Comment (vladfrangu) [inline README.md]: Just raw events

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+A Klasa plugin to handle raw events with ease.
```
Comment (vladfrangu) [inline README.md]: Use the repo's description

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+```
+
+2. Use `klasa-raw-events` in your client.
+
+```js
+const { Client } = require("klasa");
+Client.use(require("klasa-raw-events");
+```
+
+3. Create a new `raw-event` in your `raw-events` folder.
```
Comment (vladfrangu) [inline README.md]: in your `rawEvents` folder

File: README.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+
+```js
+const { RawEvent } = require("klasa-raw-events");
+
+module.exports = class extends RawEvent {
+
+    run(){
```
Comment (vladfrangu) [inline README.md]: Spacing

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (ember-ana) [reply README.md]: it's literally copied from functions lmao let me PR there too

**Comment by ember-ana** [comment]
Done & fixed for travis/markdownlint aswell as fixed typings

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**